tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -143,16 +143,10 @@ Clean Up Server
   Sleep  1
   Maybe Close Popup
   Open With JupyterLab Menu  File  New  Notebook
-  Sleep  1
-  Maybe Close Popup
-  Open With JupyterLab Menu  File  Open from Path…
-  Input Text  xpath=//input[@placeholder="/path/relative/to/jlab/root"]  Untitled.ipynb
-  Click Element  xpath://div[.="Open"]
+  Sleep  2
```

Comment:
should the Sleep be part of the "Maybe Close Popup" keyword since it seems to be needed everytime the "Maybe.." is called

Reply:
It could be, but I also don't want this PR to change more than it needs to; that can be done as part of a refactoring effort.